phpcs.xml.dist

```diff
@@ -13,7 +13,7 @@
 	<!-- Ignoring Files and Folders:
 		https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-files-and-folders -->
 	<exclude-pattern>./apigen/</exclude-pattern>
-	<exclude-pattern>./build/</exclude-pattern>
+	<exclude-pattern type="relative">build/</exclude-pattern>
```

Comment:
Curious as to what `relative` does. Does it somehow mean that it won't look in the `build` folder when running locally (which is why it was added in the first place), but still work on TravisCI? 🤞

Reply:
It should ignore the `build` directory locally (it is doing so for me when I test it). The difference is that if the parent directory for your Sensei codebase is also called `build` (as it is in the Travis environment), it will not ignore all the files. Seems like `./build` should have had the same effect, but phpcs wasn't running any checks on Travis without this fix.

Reply:
Good stuff. Thanks for fixing my mistake! 😉